h2o-algos/src/main/java/hex/generic/Generic.java

```diff
@@ -29,6 +29,7 @@
         ALLOWED_MOJO_ALGOS.add("xgboost");
         ALLOWED_MOJO_ALGOS.add("isolationforest");
         ALLOWED_MOJO_ALGOS.add("drf");
+        ALLOWED_MOJO_ALGOS.add("deeplearning");
```

Comment:
I know its not directly related to this PR but since you are touching the code: the hashSet here should definitely be unmodifiable

Reply:
Thanks for the hint, perfect time to fix this. Done.